src/docs/get-started/install/_path-mac-linux.md

```diff
@@ -44,6 +44,40 @@ whenever you open a new window. For example:
     $ which flutter
     ```
 
+### Update directly (Linux)
+
+In most cases, your distribution may not acquire the path you pass
+through the terminal for future sessions. To maintain permanently, in
```

Comment:
Replace "To maintain permanently..."  and the next sentence with:

When this occurs, you need to change the environment variables file
directly. This requires administrator privileges.